run.sh

```diff
@@ -171,21 +173,19 @@ then
     echo "❌ clang is not installed"
     ohai "Installing make and clang..."
     wait_for_user
-    sudo apt update && sudo apt upgrade -y
-    sudo apt install make clang
+    sudo apt install make clang -y
   fi
 
-  which python3 > /dev/null
+  which pip3 > /dev/null
   if [[ ($? -eq 0) ]]
   then
     python_version=`python3 -V | cut -d\  -f2`
-    echo "✅ Python ${python_version} is installed"
+    echo "✅ Python ${python_version} and pip are installed"
```

Comment:
pip noemen lijkt me hier niet per se nodig

Reply:
In de zin dat het heel erg raar is als je python zonder pip zou hebben?

Reply:
Je hebt standaard python zonder pip, daarom installeer moet je specifiek `python3-pip` installeren.

Reply:
Ik begrijp dat je dit doet omdat in feite vooral pip wordt geinstalleerd. Doe maar even zo; misschien komt er nog een betere verwoording.